submissions/adeyemiJuwon/todo/src/App.css

```diff
@@ -0,0 +1,153 @@
+/*******HEADER*********/
+.header {
+  background-color: rgb(51, 51, 131);
+  height: 300px;
+}
+.header h1 {
+  padding-top: 100px;
+}
+.header .form {
+  padding-top: 50px;
+  max-width: 500px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+/*******FORM*********/
+
+form {
+  width: 85%;
+  height: 50px;
```

Comment:
No need to set height the content  will fill it.

Reply:
Okay